### DISCUSSION TITLE
How to set public directory for assets ?

### DISCUSSION THREAD

**Discussion by bastien70** [deployphp/deployer#2305] [Help needed]
Hello, I am trying to use Deployer with Symfony 5.1

I've pretty much managed to get it to work for the time being, with one small detail.

I can't get the assets to be generated in my project's / public folder.

By default, it seems to stay on the configuration of a Symfony 3 pointing to the / web folder

Which leads me to this kind of error:

![erreur](https://user-images.githubusercontent.com/53140475/100441094-319cdc00-30a6-11eb-8524-27710e68199b.PNG)

I tried to add this in my **deploy.php**

```php
set('public_dir', 'public');
```

But no effect

**Comment by antonmedv** [comment]
What Deployer  version?

**Reply by bastien70** [reply]
It's the v.6.8.0

**Reply by antonmedv** [reply]
What is error?

**Reply by bastien70** [reply]
I've no error. Just the assets are deployed on `base_project/web` instead of the real public_directory : `base_project/public`

**Reply by antonmedv** [reply]
Try to run deploy with vvv option and spot wrong command.
